homeassistant/components/lidarr/const.py

```diff
@@ -18,7 +18,7 @@
 
 # Defaults
 DEFAULT_DAYS = "1"
-DEFAULT_HOST = "localhost"
+DEFAULT_HOST = "http://127.0.0.1:8686"
```

Comment:
This, as well as `DEFAULT_DAYS` are not used by the implementation. Please remove them